src/server/scripts/Commands/cs_modify.cpp

```diff
@@ -60,11 +60,17 @@ class modify_commandscript : public CommandScript
             { "gender",         SEC_GAMEMASTER,      false, &HandleModifyGenderCommand,        "" },
             { "speed",          SEC_GAMEMASTER,      false, nullptr,                           "", modifyspeedCommandTable }
         };
+
+        static std::vector<ChatCommand> morphCommandTable =
+        {
+            { "reset",      SEC_GAMEMASTER,     false, &HandleMorphResetCommand, "" },
+            { "target",     SEC_GAMEMASTER,     false, &HandleMorphTargetCommand, "" }
+        };
+
         static std::vector<ChatCommand> commandTable =
         {
-            { "morph",          SEC_MODERATOR,      false, &HandleModifyMorphCommand,          "" },
-            { "demorph",        SEC_MODERATOR,      false, &HandleDeMorphCommand,              "" },
-            { "modify",         SEC_GAMEMASTER,     false, nullptr,                            "", modifyCommandTable }
+            { "morph",          SEC_MODERATOR,      false, nullptr,     "", morphCommandTable },
+            { "modify",         SEC_GAMEMASTER,     false, nullptr,     "", modifyCommandTable }
```

Comment:
what is this about? (just curious if it's related to the morph)

Reply:
Modify is another command that I actually didn't re-search what it does, but morph was already inserted in here so instead of adding a new file for morph I didn't see an issue of it being here since it was already implemented.

Reply:
i think that long ago the command was .modify morph, maybe it's related and obsolete